contracts/test/MockVaultManager.sol

```diff
@@ -1,20 +1,26 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
 contract MockVaultManager {
+    address public tokenAddress;
+
     function depositByToken(
         address _tokenAddress,
         address from,
         uint _amount
     ) external returns (uint vaultId) {
-        // do nothing
+        tokenAddress = _tokenAddress;
+        IERC20(_tokenAddress).transferFrom(from, address(this), _amount);
+        vaultId = 0;
     }
 
     function withdrawByVaultId(
-        uint _vaultId,
+        uint /** _vaultId **/,
         address to,
         uint _amount
     ) external {
-        // do nothing
+        IERC20(tokenAddress).transfer(to, _amount);
```

Comment:
no need to do the actual transfer - for testing.
if the function is called - it's enough for this part

we will do integration test on other repo